src/dags/importscripts/import_financien_grootboek.py

```diff
@@ -0,0 +1,138 @@
+            FROM DATAHUB.FI2_DHB_FCT_GROOTBOEK GB
+            INNER JOIN DATAHUB.FI2_DHB_DIM_RELATIE REL ON REL.ID = GB.REL_ID
+            INNER JOIN DATAHUB.FI2_DHB_DIM_ORGANISATIE org ON ORG.ID = GB.ORG_ID
+            INNER JOIN DATAHUB.FI2_DHB_DIM_GROOTBOEKREKENING gbr ON gbr.ID = GB.gbr_ID
+            INNER JOIN DATAHUB.FI2_DHB_DIM_VERPLICHTINGREGEL vpr ON vpr.ID = GB.VPR_ID
+            INNER JOIN DATAHUB.FI2_DHB_DIM_WERKORDER wor ON wor.ID = GB.WOR_ID
+            WHERE 1=1
+            AND RVE_CODE = '380000'
+            """,
+            connection,
+            coerce_float=True,
+            params=None,
+            parse_dates={
+                "datum_boeking": date_fmt,
+                "vpr_datum_grootboek": date_fmt,
+                "vpr_datumverplichting": date_fmt,
+                "vpr_dtm_lste_afboeking": date_fmt,
+            },
+            columns=None,
+            chunksize=None,
+        )
+        dtype = {
+            "datum_boeking": Date(),
+            "bedrag_boeking": Numeric(),
+            "boeking_omschrijving": Text(),
+            "boeking_opmerking": Text(),
+            "hoofdrekening_code": Text(),
+            "rekening_omschrijving": Text(),
+            "bedrijfseenheid_code": Text(),
+            "bedrijfseenheid_omschrijving": Text(),
+            "subgrootboek": Text(),
+            "subgrootboektype": Text(),
+            "vpl_verplichtingnummer": Text(),
+            "vpl_omschrijving": Text(),
+            "vpl_jaar": Integer(),
+            "vpl_inkoopcontractcode": Text(),
+            "vpl_status": Text(),
+            "vpr_datum_laatste_afboeking": Date(),
+            "vpr_bedrag_oorspronkelijk": Numeric(),
+            "vpr_bedrag_afgeboekt": Numeric(),
+            "vpr_bedrag_restant": Numeric(),
+            "vpr_regelnummer": Integer(),
+            "vpr_omschrijving": Text(),
+            "vpr_opmerking": Text(),
+            "vpr_datum_grootboek": Date(),
+            "vpr_datumverplichting": Date(),
+            "werkorder_code": Text(),
+            "werkorder_omschrijving": Text(),
+            "werkorder_status": Text(),
+            "werkorder_status_opmerking": Text(),
+            "relatie_code": Text(),
+            "relatie_naam": Text(),
+            "indicatie_crediteur": Boolean(),
+            "indicatie_debiteur": Boolean(),
+            "subteam_code": Text(),
+            "subteam_naam": Text(),
+            "team_code": Text(),
+            "team_naam": Text(),
+            "afdeling_code": Text(),
+            "afdeling_naam": Text(),
+            "rve_code": Text(),
+            "rve_naam": Text(),
+            "cluster_code": Text(),
+            "cluster_naam": Text(),
+        }
+        df.to_sql(
+            table_name, db_engine, if_exists="replace", index_label="id", index=True, dtype=dtype
+        )
+        with db_engine.connect() as connection:
+            connection.execute(f"ALTER TABLE {table_name} ADD PRIMARY KEY (ID)")
```

Comment:
We should not be using regular string formatting for constructing SQL queries. It's probably safe here, but the practice gets copied over and over and as such persists. Better to use the features of `psycopg2`: https://www.psycopg.org/docs/sql.html

Using that the statement would become (untested):
```python
from psycopg2 import sql
...
connection.execute(
    sql.SQL("ALTER TABLE {table_name} ADD PRIMARY KEY (ID)")
        .format(table_name=sql.Identifier(table_name)
)
```